app/models/customer.rb

```diff
@@ -75,6 +82,18 @@ def self.ransackable_attributes(_auth_object = nil)
     %w[id name external_id email]
   end
 
+  def display_name(prefer_legal_name: false)
```

Comment:
We don't need this attribute after the discussion in the dive-in we decided to always prefer legal_name.

```suggestion
  def display_name
```